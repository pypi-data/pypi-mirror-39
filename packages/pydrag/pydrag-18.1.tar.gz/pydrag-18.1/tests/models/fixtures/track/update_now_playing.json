{
    "interactions": [
        {
            "request": {
                "body": "method=auth.getMobileSession&format=json&api_key=LAST_FM_API_KEY&username=Zaratoustre&authToken=USER_AUTH_TOKEN&api_sig=API_SIG",
                "headers": {
                    "Accept": [
                        "*/*"
                    ],
                    "Accept-Encoding": [
                        "gzip, deflate"
                    ],
                    "Connection": [
                        "keep-alive"
                    ],
                    "Content-Length": [
                        "186"
                    ],
                    "Content-Type": [
                        "application/x-www-form-urlencoded"
                    ],
                    "User-Agent": [
                        "python-requests/2.20.0"
                    ]
                },
                "method": "POST",
                "uri": "https://ws.audioscrobbler.com/2.0/"
            },
            "response": {
                "body": {
                    "string": "{\"session\":{\"subscriber\":0,\"name\":\"Zaratoustre\",\"key\":\"CENSORED\"}}"
                },
                "headers": {
                    "Access-Control-Allow-Methods": [
                        "POST, GET, OPTIONS"
                    ],
                    "Access-Control-Allow-Origin": [
                        "*"
                    ],
                    "Access-Control-Max-Age": [
                        "86400"
                    ],
                    "Connection": [
                        "keep-alive"
                    ],
                    "Content-Type": [
                        "application/json"
                    ],
                    "Date": [
                        "Sat, 10 Nov 2018 18:45:58 GMT"
                    ],
                    "Server": [
                        "openresty/1.9.7.5"
                    ],
                    "content-length": [
                        "90"
                    ]
                },
                "status": {
                    "code": 200,
                    "message": "OK"
                }
            }
        },
        {
            "request": {
                "body": "artist=AC%2FDC&track=Hells+Bells&trackNumber=2&sk=USER_SESSION&method=track.updateNowPlaying&format=json&api_key=LAST_FM_API_KEY&api_sig=API_SIG",
                "headers": {
                    "Accept": [
                        "*/*"
                    ],
                    "Accept-Encoding": [
                        "gzip, deflate"
                    ],
                    "Connection": [
                        "keep-alive"
                    ],
                    "Content-Length": [
                        "206"
                    ],
                    "Content-Type": [
                        "application/x-www-form-urlencoded"
                    ],
                    "User-Agent": [
                        "python-requests/2.20.0"
                    ]
                },
                "method": "POST",
                "uri": "https://ws.audioscrobbler.com/2.0/"
            },
            "response": {
                "body": {
                    "string": "{\"nowplaying\":{\"artist\":{\"corrected\":\"0\",\"#text\":\"AC\\/DC\"},\"ignoredMessage\":{\"code\":\"0\",\"#text\":\"\"},\"album\":{\"corrected\":\"0\",\"#text\":\"\"},\"albumArtist\":{\"corrected\":\"0\",\"#text\":\"\"},\"track\":{\"corrected\":\"0\",\"#text\":\"Hells Bells\"}}}"
                },
                "headers": {
                    "Access-Control-Allow-Methods": [
                        "POST, GET, OPTIONS"
                    ],
                    "Access-Control-Allow-Origin": [
                        "*"
                    ],
                    "Access-Control-Max-Age": [
                        "86400"
                    ],
                    "Connection": [
                        "keep-alive"
                    ],
                    "Content-Type": [
                        "application/json"
                    ],
                    "Date": [
                        "Sat, 10 Nov 2018 18:45:59 GMT"
                    ],
                    "Server": [
                        "openresty/1.9.7.5"
                    ],
                    "content-length": [
                        "229"
                    ]
                },
                "status": {
                    "code": 200,
                    "message": "OK"
                }
            }
        }
    ],
    "version": 1
}
