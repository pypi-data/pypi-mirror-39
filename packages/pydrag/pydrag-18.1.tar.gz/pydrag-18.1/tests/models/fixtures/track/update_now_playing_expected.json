{
    "artist": "AC/DC",
    "timestamp": 11111111111,
    "track": "Hells Bells"
}
